packages/web/src/components/browser-push-confirmation-modal/BrowserPushConfirmationModal.tsx

```diff
@@ -141,18 +142,22 @@ const ConnectedBrowserPushConfirmationModal = ({
         <div className={styles.buttons}>
           <Button
             className={styles.closeButton}
-            text={messages.close.toUpperCase()}
-            size={ButtonSize.MEDIUM}
-            type={ButtonType.COMMON}
+            variant='secondary'
+            size='default'
             onClick={onClose}
-          />
+            css={{ fontSize: 16 }}
+          >
+            {messages.close.toUpperCase()}
+          </Button>
           <Button
             className={styles.enableButton}
-            text={messages.confirm.toUpperCase()}
-            size={ButtonSize.MEDIUM}
-            type={ButtonType.PRIMARY_ALT}
+            variant='primary'
+            size='default'
             onClick={onEnabled}
-          />
+            css={{ fontSize: 16 }}
```

Comment:
any chance we can avoid this? additionally the uppercase changes? looks like only our large buttons allow this. If we must do uppercase, should prob build into component, and at minimum maybe we do `textTransform: uppercase` custom style?

Reply:
yeah definitely. this component is weird, the buttons look super whack without this custom font size. since it's a key part of the first time user flow i didn't want to change it without consulting design, but we can revisit. I'll put it in the linear